src/pages/home/sidebar/OptionRow.js

```diff
@@ -110,11 +110,14 @@ const OptionRow = ({
         ? hoverStyle.backgroundColor
         : backgroundColor;
     const focusedBackgroundColor = styles.sidebarLinkActive.backgroundColor;
+    const isMultipleParticipant = option.participantsList.length > 1;
```

Comment:
This line in particular caused a regression, because the `option.participantsList` may be undefined per the prop-types [here](https://github.com/Expensify/Expensify.cash/blob/15f9b08595a975a779d84d298b7ae32830aa2e68/src/pages/home/sidebar/optionPropTypes.js#L22)

Reply:
may be:  const isMultipleParticipant = option.participantsList?.length  > 1